samza-test/src/test/java/org/apache/samza/test/samzasql/TestSamzaSqlEndToEnd.java

```diff
@@ -110,6 +130,28 @@ public void testEndToEndMultiSqlStmts() {
     Assert.assertTrue(IntStream.range(0, numMessages).boxed().collect(Collectors.toList()).equals(new ArrayList<>(outMessagesSet)));
   }
 
+  @Ignore
+  @Test
+  public void testEndToEndMultiSqlStmtsWithSameSystemStreamAsInputAndOutput() {
```

Comment:
Thanks for adding this test. Can you add a comment on why we are ignoring this and possibly point to the Jira ticket tracking the issue?

Reply:
Ok